tests/resources/mir_datasets/compmusic_otmm_makam/MTG-otmm_makam_recognition_dataset-f14c0d0/extras/wrapper_melody.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from morty.extras.pitch import Pitch
+import os
+
+folder = os.path.join("..", "data")
+
+p = Pitch()
+p.extract(folder)
```

Comment:
Is this useful to port this script to `mirdata`?

Reply:
It won't be used for the mirdata loader, but since the whole set of dataset files is compressed in a single .zip file, when retrieving the data from Zenodo this script file is also downloaded and extracted from the .zip file (same happens for `metadata.py`. I decided not to write an extra download function to delete this script once downloaded (it may even be useful for the dataset users). However, it was my bad to add it into the testing resources, because it is not needed and it is unnecessarily occupying space. Same for `metadata.py`. Will remove them, thanks for the catch!